src/main/app/address/ManualAddressPostControllerBase.ts

```diff
@@ -0,0 +1,43 @@
+import autobind from 'autobind-decorator';
```

Comment:
Recommended to use to-api-format.ts for such prefix mapping instead of creating new controllers

Reply:
I created new controller as the default controllers were overriding the returnUrl value and I didn't want to change them, the value I needed to add comes from req.query, how could I do that in to-api-format.ts?